Test Failure: com.ibm.ws.concurrent.persistent.fat.errorpaths.PersistentExecutorErrorPathsTest.testTransactionTimeout

```
testTransactionTimeout:junit.framework.AssertionFailedError: 2019-10-30-12:00:08:325 Missing success message in output. PersistentErrorTestServlet is starting testTransactionTimeout<br>
<pre>ERROR in testTransactionTimeout:
java.lang.IllegalStateException: CWWKC1551E: You can invoke the get method only on TaskStatus that is obtained after the task has ended. Use the getResult method to obtain the result of the execution that was the most recent execution at the time when the TaskStatus instance was obtained.
at com.ibm.ws.concurrent.persistent.internal.TaskStatusImpl.get(TaskStatusImpl.java:179)
at web.PersistentErrorTestServlet.testTransactionTimeout(PersistentErrorTestServlet.java:1013)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at web.PersistentErrorTestServlet.doGet(PersistentErrorTestServlet.java:89)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1230)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:729)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:426)
at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1218)
at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1002)
at com.ibm.ws.webcontainer.servlet.CacheServletWrapper.handleRequest(CacheServletWrapper.java:75)
at com.ibm.ws.webcontainer.WebContainer.handleRequest(WebContainer.java:938)
at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost$2.run(DynamicVirtualHost.java:279)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink$TaskWrapper.run(HttpDispatc
```

createEntityManager() from
```
        EntityManager em = getPersistenceServiceUnit().createEntityManager();
```
is taking over 50 seconds to complete with an exception indicating the expected error for a timed out transaction (the test case sets transaction timeout to 1 second and delays 10 in order to force a transaction timeout).  This 50+ seconds in combination with the 10 exceeds the 1 minute that the test was willing to wait, and it fails.

```
[10/30/19, 11:59:28:200 GMT] 00000076 id=8f06d575 com.ibm.ws.concurrent.persistent.db.DatabaseTaskStore        > persist Entry  
                                                                                                               TaskRecord@b91f2687
PREVSCHD=2019/10/30-11:59:08.122-GMT[1572436748122]
PREVSTRT=2019/10/30-11:59:18.200-GMT[1572436758200]
PREVSTOP=2019/10/30-11:59:28.200-GMT[1572436768200]
RESULT=byte[78]
FAILURES=0
STATE=ENDED
                                                                                                               TaskRecord[21]@86771322
VERSION=0
                                                                                                               UPDATE Task t SET t.PREVSCHED=:sc2,t.PREVSTART=:sa2,t.PREVSTOP=:so2,t.RESLT=:r2,t.RFAILS=:f2,t.STATES=:s2,t.VERSION=t.VERSION+1 WHERE t.ID=:i AND t.VERSION=:v1
[10/30/19, 11:59:28:200 GMT] 00000076 id=8f06d575 com.ibm.ws.concurrent.persistent.db.DatabaseTaskStore        > getPersistenceServiceUnit Entry 
[10/30/19, 11:59:28:200 GMT] 00000076 id=8f06d575 com.ibm.ws.concurrent.persistent.db.DatabaseTaskStore        < getPersistenceServiceUnit Exit  
                                                                                                               com.ibm.wsspi.persistence.internal.PersistenceServiceUnitImpl@1f4bc5a3
...
[10/30/19, 12:00:23:215 GMT] 00000076 id=10d774eb com.ibm.ws.concurrent.persistent.internal.InvokerTask        3 marking transaction to roll back in response to error 
                                                                                                               javax.persistence.PersistenceException: 
Exception Description: Error binding to externally managed transaction
Internal Exception: javax.transaction.RollbackException: Transaction rolled back
	at org.eclipse.persistence.internal.jpa.transaction.JTATransactionWrapper.registerIfRequired(JTATransactionWrapper.java:145)
	at org.eclipse.persistence.internal.jpa.EntityManagerImpl.setJTATransactionWrapper(EntityManagerImpl.java:2168)
	at org.eclipse.persistence.internal.jpa.EntityManagerImpl.detectTransactionWrapper(EntityManagerImpl.java:893)
	at org.eclipse.persistence.internal.jpa.EntityManagerImpl.initialize(EntityManagerImpl.java:416)
	at org.eclipse.persistence.internal.jpa.EntityManagerImpl.<init>(EntityManagerImpl.java:409)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:324)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:337)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:303)
	at com.ibm.wsspi.persistence.internal.PersistenceServiceUnitImpl.createEntityManager(PersistenceServiceUnitImpl.java:96)
	at com.ibm.ws.concurrent.persistent.db.DatabaseTaskStore.persist(DatabaseTaskStore.java:1286)
	at com.ibm.ws.concurrent.persistent.internal.InvokerTask.run(InvokerTask.java:517)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:239)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:825)
Caused by: javax.transaction.RollbackException: Transaction rolled back
	at com.ibm.tx.jta.impl.TransactionImpl.registerSynchronization(TransactionImpl.java:2285)
	at com.ibm.tx.jta.impl.TransactionImpl.registerSynchronization(TransactionImpl.java:2265)
	at org.eclipse.persistence.internal.jpa.transaction.JTATransactionWrapper.registerIfRequired(JTATransactionWrapper.java:136)
	... 16 more
```

Although this already seems excessive, most of the test buckets allow up to 2 minutes, so I'll update this one with the additional tolerance.

--- a/dev/com.ibm.ws.concurrent.persistent_fat_errorpaths/test-applications/persistenterrtest/src/web/PersistentErrorTestServlet.java
+++ b/dev/com.ibm.ws.concurrent.persistent_fat_errorpaths/test-applications/persistenterrtest/src/web/PersistentErrorTestServlet.java
@@ -64,7 +64,7 @@ public class PersistentErrorTestServlet extends HttpServlet {
     /**
      * Maximum number of nanoseconds to wait for a task to finish.
      */
-    private static final long TIMEOUT_NS = TimeUnit.MINUTES.toNanos(1);
+    private static final long TIMEOUT_NS = TimeUnit.MINUTES.toNanos(2);
 
     @Resource(name = "java:comp/env/concurrent/mySchedulerRef", lookup = "concurrent/myScheduler")
     private PersistentExecutor scheduler;

